mygrad/nnet/layers/gru.py

```diff
@@ -438,112 +422,117 @@ def gru(
+    bh : array_like, shape=(D,)
+       The biases used to scale a hidden-descriptor.
 
-        s0 : Optional[array_like], shape=(N, D)
-            The 'seed' hidden descriptors to feed into the RNN. If None, a Tensor
-            of zeros of shape (N, D) is created.
+    s0 : Optional[array_like], shape=(N, D)
+        The 'seed' hidden descriptors to feed into the RNN. If None, a Tensor
+        of zeros of shape (N, D) is created.
 
-        bp_lim : Optional[int]
-            The (non-zero) limit of the depth of back propagation through time to be
-            performed. If `None` back propagation is passed back through the entire sequence.
+    bp_lim : Optional[int]
+        *This feature is experimental and is currently untested*.
+        The (non-zero) limit of the depth of back propagation through time to be
+        performed. If `None` back propagation is passed back through the entire sequence.
 
-            E.g. `bp_lim=3` will propagate gradients only up to 3 steps backward through the
-            recursive sequence.
+        E.g. `bp_lim=3` will propagate gradients only up to 3 steps backward through the
+        recursive sequence.
 
-        dropout : float (default=0.), 0 <= dropout < 1
-            If non-zero, the dropout scheme described in [1]_ is applied. See Notes
-            for more details.
+    dropout : float (default=0.), 0 <= dropout < 1
+        If non-zero, the dropout scheme described in [1]_ is applied. See Notes
+        for more details.
 
-        constant : bool, optional (default=False)
-            If True, the resulting Tensor is a constant.
+    constant : bool, optional (default=False)
+        If True, the resulting Tensor is a constant.
 
-        Returns
-        -------
-        mygrad.Tensor, shape=(T+1, N, D)
-            The sequence of 'hidden-descriptors' produced by the forward pass of the RNN.
+    Returns
+    -------
+    mygrad.Tensor, shape=(T+1, N, D)
+        The sequence of 'hidden-descriptors' produced by the forward pass of the RNN.
 
-        Notes
-        -----
-        - :math:`T` : Sequence length
-        - :math:`N` : Batch size
-        - :math:`C` : Length of single datum
-        - :math:`D` : Length of 'hidden' descriptor
+    Notes
+    -----
+    - :math:`T` : Sequence length
+    - :math:`N` : Batch size
+    - :math:`C` : Length of single datum
+    - :math:`D` : Length of 'hidden' descriptor
 
-        The GRU system of equations is given by:
+    The GRU system of equations is given by:
 
-        .. math::
+    .. math::
 
-                    Z_{t} = \sigma (X_{t} U_z + S_{t-1} Wz + bz)
+                Z_{t} = \sigma (X_{t} U_z + S_{t-1} Wz + bz)
 
-                    R_{t} = \sigma (X_{t} U_r + S_{t-1} Wr + br)
+                R_{t} = \sigma (X_{t} U_r + S_{t-1} Wr + br)
 
-                    H_{t} =  tanh(X_{t} U_h + (R_{t} * S_{t-1}) W_h + b_h)
+                H_{t} =  tanh(X_{t} U_h + (R_{t} * S_{t-1}) W_h + b_h)
```

Comment:
Update equation formatting to be consistent with that at top of docstring
```suggestion
                H_{t} =    tanh(X_{t} U_h + (R_{t} * S_{t-1}) W_h + b_h)
```